client/src/js/services/PrototypeApiService.js

```diff
@@ -161,4 +163,32 @@ function PrototypeApiService($http, util) {
     return this.$http.delete(target)
       .then(util.unwrapHttpResponse);
   }
+
+  /**
+   * @method reports
+   *
+   * @description
+   * Sends an HTTP GET request to the url "/route/reports/:id"
+   * to get a document as a report.
+   *
+   * @param {Number} id - the identifier of the URL route.
```

Comment:
Should this always be a number?  Would it make more sense to have `/reports/inventory/expired`, and so on, instead of `/reports/1`?